README.rst

```diff
@@ -55,7 +55,11 @@ and pull requests at `Aiven examples repository <https://github.com/aiven/aiven-
 
 Trademark
 ---------
-Apache Kafka and Apache Cassandra are either registered trademarks or trademarks of the Apache Software Foundation in the United States and/or other countries. M3, OpenSearch, Elasticsearch, PostgreSQL, Redis, InfluxDB, Grafana are trademarks and property of their respective owners. All product and service names used in this website are for identification purposes only and do not imply endorsement. 
+Apache Kafka and Apache Cassandra are either registered trademarks or trademarks of the Apache Software Foundation in the United States and/or other countries.
+
+\* Redis is a trademark of Redis Ltd. Any rights therein are reserved to Redis Ltd. Any use by Aiven Oy is for referential purposes only and does not indicate any sponsorship, endorsement or affiliation between Redis and Aiven Oy.
+
+M3, OpenSearch, PostgreSQL, InfluxDB, Grafana are trademarks and property of their respective owners. All product and service names used in this website are for identification purposes only and do not imply endorsement. Elasticsearch is a trademark of Elasticsearch BV, registered in the U.S. and in other countries.
```

Comment:
LGTM
the only question is: does it make sense to have a statement about Elasticsearch on the same line or should it be on a different one since it has a separate clarification.

Reply:
I think it's fine this way, the Redis one needs to be visible, but I guess this is cosmetics. I'll merge this one

Reply:
After you approve of course